akka-docs/src/test/java/jdocs/typed/tutorial_5/Device.java

```diff
@@ -53,10 +53,12 @@ public ReadTemperature(long requestId, ActorRef<RespondTemperature> replyTo) {
     final String deviceId;
     final Optional<Double> value;
 
+    // #respond-declare
     public RespondTemperature(long requestId, String deviceId, Optional<Double> value) {
       this.requestId = requestId;
       this.deviceId = deviceId;
       this.value = value;
+      // #respond-declare
```

Comment:
I think this looked a bit weird in the page, with only pulling in the constructor, let's make it the entire class for Java.

Reply:
Made the change, how's this?

Reply:
Sorry, I should have been more clear in my feedback. 

I meant that `// #respond-declare`  should be the entire class `RespondTemperature` in Java, just like it is in Scala. I think it is fine like it was before except that.

Reply:
No worries, my Java skills are not all that so I failed to see the class borders.